src/main/java/org/betonquest/betonquest/id/ScheduleID.java

```diff
@@ -0,0 +1,38 @@
+package org.betonquest.betonquest.id;
```

Comment:
Should we may start moving the IDs to the related code. This would mean that this should be in `org.betonquest.betonquest.api.schedule`?